mkt/api/base.py

```diff
@@ -172,6 +172,16 @@ def get_region_from_request(self, request):
         """
         return get_region_from_request(request)
 
+    def get_serializer_class(self):
+        """
+        Return a serializer for the requested API version. Versioned
+        serializers are registered on the default serializer with their version
+        number.
+        """
+        api_version = 'V{v_int}'.format(v_int=self.request.API_VERSION)
```

Comment:
Only reason was it looks nicer in a camel case class name `AppSerializerV1` is more obvious than `AppSerializerv1` and then wanted to be consistent.